data/net/minecraft/advancements/critereon/LightningStrikeTrigger.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/advancements/critereon/LightningStrikeTrigger
+	METHOD lambda$trigger$0 (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/entity/Entity;)Lnet/minecraft/world/level/storage/loot/LootContext;
+		ARG 1 entity
+	METHOD lambda$trigger$1 (Lnet/minecraft/world/level/storage/loot/LootContext;Ljava/util/List;Lnet/minecraft/advancements/critereon/LightningStrikeTrigger$TriggerInstance;)Z
+		ARG 2 instance
+	METHOD trigger (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/entity/LightningBolt;Ljava/util/List;)V
+		ARG 1 player
+		ARG 2 lightning
+		ARG 3 nearbyEntities
+	CLASS TriggerInstance
+		METHOD <init> (Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;)V
+			ARG 1 player
+			ARG 2 lightning
+			ARG 3 bystander
+		METHOD lighthingStrike (Lnet/minecraft/advancements/critereon/EntityPredicate;Lnet/minecraft/advancements/critereon/EntityPredicate;)Lnet/minecraft/advancements/critereon/LightningStrikeTrigger$TriggerInstance;
+			ARG 0 lightningPredicate
+			ARG 1 bystanderPredicate
+		METHOD matches (Lnet/minecraft/world/level/storage/loot/LootContext;Ljava/util/List;)Z
+			ARG 1 playerContext
+			ARG 2 entitiesContext
```

Comment:
I think this name is slightly inaccurate. I'm thinking something more like `entityContexts` to reflect individual loot contexts per entity but plural because of the list.